bin/steps/python

```diff
@@ -27,6 +27,14 @@ if [[ "$STACK" != "$CACHED_PYTHON_STACK" ]]; then
     rm -fr .heroku/python-stack .heroku/python-version .heroku/python .heroku/vendor
```

Comment:
I think `.heroku/python-sqlite3-version` needs to be added to the list of files deleted here, since otherwise if the stack changes, the sqlite binary and headers end up being deleted, but yet the `bin/steps/sqlite3` won't reinstall them, since `.heroku/python-sqlite3-version` will still exist.

Alternatively, perhaps the sqlite3 version file could be moved into `.heroku/python/`, since they are quite strongly tied together?

Reply:
Good catch. I kept it out to keep it in line with `.heroku/python-version`.